ci/qa/phpmd.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<ruleset name="Ibuildings QA Tools Default Ruleset"
+         xmlns="http://pmd.sf.net/ruleset/1.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://pmd.sf.net/ruleset/1.0.0
+                     http://pmd.sf.net/ruleset_xml_schema.xsd"
+         xsi:noNamespaceSchemaLocation="
+                     http://pmd.sf.net/ruleset_xml_schema.xsd">
+    <description>
+        Ibuildings QA Tools Default Ruleset
+    </description>
+
+    <rule ref="rulesets/cleancode.xml">
+        <exclude name="BooleanArgumentFlag" />
+        <exclude name="ElseExpression" />
+        <exclude name="StaticAccess" />
+    </rule>
+
+    <rule ref="rulesets/unusedcode.xml">
+        <!-- we turn this back on as soon as an exception for inherited/implemented methods is available-->
+        <exclude name="UnusedFormalParameter"/>
+    </rule>
+
+    <rule ref="rulesets/codesize.xml" />
+
+    <rule ref="rulesets/controversial.xml" />
+
+    <rule ref="rulesets/design.xml" />
+
+    <rule ref="rulesets/naming.xml">
+        <exclude name="ShortVariable" />
+        <exclude name="LongVariable" />
+    </rule>
+
+</ruleset>
```

Comment:
I'd expect a move of the /phpmd.xml file. That way you preserve the version history of the file! If you'd like I can show you how.